biothings/web/query/builder.py

```diff
@@ -360,10 +357,19 @@ def apply_extras(self, search, options):
         # https://www.elastic.co/guide/en/elasticsearch/
         # reference/current/index-modules.html
 
+        # Feature: post_filter
+        # -- implementation using query string matching
+        # Ref: https://www.elastic.co/guide/en/elasticsearch/reference/8.1/filter-search-results.html#post-filter
+        if options.aggs and options.post_filter:
```

Comment:
@sengineer0 can `post_filter` work without `aggs`? Please verify it on ES.

If it works without `aggs`, will `post_filter` changes the `total` value in the response?

Reply:
ok, let's me check it

Reply:
I tried to remove `aggs`, and the `post_filter` still worked, and it changed `total` value.
Although the ES's documentation didn't tell that `post_filter` must be used with `aggs`, I am prefer to only use `post_filter` with `aggs`.

Reply:
@newgene

Reply:
ok, thanks for confirming. In this case, let's remove the `aggs` requirements. I agree it's typically used together with `aggs`, but I think we have a use case for `post_filter` without `aggs` as well.

Reply:
ok. no problem with me. I will update my PR.

Reply:
It's updated. @newgene